content/02.body.md

```diff
@@ -0,0 +1,282 @@
+While systematic methods are not free of all bias, the bias introduced should be independent of degree.
+If the biases of high-throughput experimentation are roughly degree-independent, then the degree distributions of networks produced via comprehensive, systematic experimentation resemble the degree distributions of true relationships.
+When such unbiased data are available, the degree bias in literature-derived networks can be quantified by comparing degree between literature-derived and unbiased networks.
+
+Figure {@fig:node-degree-bias} gives examples of degree distribution bias in five networks, two protein-protein interaction networks (one of which is literature-derived, and the other of which was constructed via a high-throughput screen of all human proteins), two transcription factor-target gene networks (one of which is literature-derived, and the other of which was constructed from a ChIP-seq screen), and a co-authorship network based on bioinformatics preprints posted on bioRxiv. 
+Figure {@fig:sampling_vs_systematic} shows bias in individual node degree for the PPI and TF-TG networks. 
+On the one hand, when edges are uniformly sampled there remains a linear correlation between degree in the networks. 
+On the other hand, non-random sampling--as is done in low-throughput scientific experimentation--can introduce bias in both directions relative to systematic investigations.
+
+![Degree distributions of biased and unbiased networks can be very different. Protein-protein interaction (PPI) literature-derived network from STRING (Szklarczyk et al.) [@doi:10.1093/nar/gky1131], high-throughput-derived networks from Rual et al. [@doi:10.1038/nature04209] and Rolland et al. [@doi:10.1016/j.cell.2014.10.050]. Transcription factor-target gene (TF-TG) literature-derived network from Han et al. [@doi:10.1093/nar/gkx1013] and high-throughput network from Lachmann et al. [@doi:10.1093/bioinformatics/btq466].  BioRxiv data from Rxivist [@doi:10.1101/515643; @doi:10.5281/zenodo.2566421].](https://github.com/greenelab/xswap-analysis/raw/bd40eb244ffe37454dc08099f5742ef3e962c70d/img/1_degree_bias/histogram.png){#fig:node-degree-bias width="100%}
+
+![Individual node degree by sampling method. In the left column, 70% of literature edges were sampled with uniform probability. In the right column, literature derived degrees are over or under the corresponding systematically-derived degree.](https://github.com/greenelab/xswap-analysis/raw/bd40eb244ffe37454dc08099f5742ef3e962c70d/img/7_well_studied_degree_bias/hexbin.png){#fig:sampling_vs_systematic width="85%"}
+
+Networks created through degree-unbiased experimentation have node degrees that are linearly correlated with the number of true relationships.
+Many domains, however, do not have unbiased experimental methods, either because such methods have not yet been developed or because such methods are not possible, in principle.
```

Comment:
```suggestion
Many domains, however, do not have unbiased experimental methods, either because such methods have not yet been developed or because such methods are not possible.
```